### ISSUE TITLE
Doesn't calculate azimuthal shear beyond 100 km range

### ISSUE THREAD

**Issue by syedhamidali** [jordanbrook/PyMeso#6]
Dear Jordan, 

I tried to calculate azimuthal shear however it doesn't seem correct to me. The issue is that it is only plotting up to 100km only. Kindly Have a look at this notebook https://github.com/syedhamidali/CAPPI-NETCDF/blob/main/AZI_SHEAR_TEST.ipynb
Here is the plot;
![image](https://user-images.githubusercontent.com/35923822/175756057-66287ea6-7330-4b42-bc72-20379dee569b.png)

**Comment by jordanbrook** [comment]
Hi Syed, 

The maximum range of the az shear field is limited by the window size that the LLSD method is calculated over. Refer to the attached document on page 12 for an explanation. The window size was originally hard-coded in the llsd_compute function, but I've just merged a pull request to allow the user to control the window size in the llsd.main() function. Refer to the updated notebook for an example of how you can use the new parameter. In short: increasing the azimuthal window size (the second value in the new window_size tuple), allows you to retrieve LLSD out to a greater range, however it also smooths out the az_shear values. I would recommend experimenting with the values to find out what works best for you. 

Cheers, 
Jordan

[Adapting_an_Australian_Mesocyclone_Detection_Algorithm.pdf](https://github.com/jordanbrook/PyMeso/files/8983965/Adapting_an_Australian_Mesocyclone_Detection_Algorithm.pdf)

**Comment by syedhamidali** [comment]
Hi Jordan,

Thanks for this quick solution and explanation. I also want to ask about `pymeso.vortex` function. Is it only an example, or can we calculate tangential velocity using that?

**Comment by jordanbrook** [comment]
Hi Syed, the tangential velocity you're referring to is for an idealized Rankine vortex, we used these vortices to test the azimuthal shear calculations (see page 5 of the document I linked above for more details). 

If you had the full horizontal wind components (U and V) you would be able to calculate the tangential velocity for real radar data, however in practice we only get the Doppler measurements. This is why we have to rely on approximations like az-shear to measure rotation. Hope that helps! Cheers.
